types/mithril/test/test-stream.ts

```diff
@@ -215,112 +204,6 @@ import { Stream } from 'mithril/stream';
 	console.assert(doubled() === 4);
 }
 
-{
```

Comment:
Why were these removed?

Reply:
Those tests were copied/adapted from Mithril's own test suite, which are used to test that the fantasy land method names (which are there for spec compliance) work.

We have not added type definitions for the quoted methods (in general, users use `stream.map` not `stream["fantasy-land/map"]`, etc.)

Because the DT rules now prevent us from allowing any index types, we can't keep these tests without adding those types. And those tests weren't really useful as type tests previously.

Reply:
Thanks for the clarification!